examples/ganache/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@reach-sh/ganache",
+  "type": "module",
+  "dependencies": {
+    "ethers": "^5.0.19",
+    "ganache-core": "2.13.2"
```

Comment:
I think, but could be wrong, that only this file is needed and we can remove the Makefile, Dockerfile, and docker-compose and they will be picked up by `reach run`